src/SpecialFunctions.jl

```diff
@@ -76,4 +76,11 @@ include("sincosint.jl")
 include("gamma.jl")
 include("deprecated.jl")
 
+@static if isdefined(Base, :missing)
+    for f in (:beta, :digamma, :erf, :erfc, :erfcinv, :erfcx, :erfi, :erfinv, :eta, :gamma,
```

Comment:
We'll have to structure things a little differently for some of these. In particular, `beta` takes two arguments, and we'd want the result to be missing if either or both is missing, e.g.
```julia
beta(a::Missing, b::Number) = missing
beta(a::Number, b::Missing) = missing
beta(a::Missing, b::Missing) = missing
```
Further, `polygamma(m, x)` is the `m`th derivative of `lgamma(x)`, so we should instead define
```julia
polygamma(m::Integer, x::Missing) = missing
```
but don't let `m` be missing.